coalib/processes/Processing.py

```diff
@@ -259,6 +259,8 @@ def get_file_dict(filename_list, log_printer):
                                       exception,
                                       log_level=LOG_LEVEL.WARNING)
 
+    for filename in file_dict.keys():
+        log_printer.debug(filename)
```

Comment:
Don't we want to have a message that these are checked files? Also, we should have one constructed debug message, not several.

This is how it looks right now, 

```
[DEBUG][11:48:45] /home/sanket/coala/docs/Users/Tutorials/Tutorial.rst
[DEBUG][11:48:45] /home/sanket/coala/docs/Users/Git_Hooks.rst
[DEBUG][11:48:45] /home/sanket/coala/docs/Users/Exit_Codes.rst
[DEBUG][11:48:45] /home/sanket/coala/docs/Getting_Involved/Writing_Documentation.rst
[DEBUG][11:48:45] /home/sanket/coala/docs/Users/Install.rst
[DEBUG][11:48:45] /home/sanket/coala/docs/Getting_Involved/MAC_Hints.rst
[DEBUG][11:48:45] /home/sanket/coala/docs/Getting_Involved/Review.rst
[DEBUG][11:48:45] /home/sanket/coala/docs/Users/Tutorials/Testing_Bears.rst
[DEBUG][11:48:45] /home/sanket/coala/docs/Users/coafile.rst
[DEBUG][11:48:45] /home/sanket/coala/docs/Getting_Involved/Writing_Tests.rst
[DEBUG][11:48:45] /home/sanket/coala/docs/Users/Tutorials/Git_Help.rst
[DEBUG][11:48:45] /home/sanket/coala/docs/General_Dev_Info/git_tutorial_1.rst
[DEBUG][11:48:45] /home/sanket/coala/docs/Getting_Involved/Writing_Good_Commits.rst
[DEBUG][11:48:45] /home/sanket/coala/docs/Users/Tutorials/Self_Correcting_Bears.rst
[DEBUG][11:48:45] /home/sanket/coala/docs/index.rst
[DEBUG][11:48:45] /home/sanket/coala/README.rst
[DEBUG][11:48:45] /home/sanket/coala/docs/Getting_Involved/Newcomers.rst
[DEBUG][11:48:45] /home/sanket/coala/docs/Users/Tutorials/Linter_Bears.rst
[DEBUG][11:48:45] /home/sanket/coala/docs/Getting_Involved/Codestyle.rst
[DEBUG][11:48:45] /home/sanket/coala/docs/Users/Glob_Patterns.rst
[DEBUG][11:48:45] /home/sanket/coala/docs/Getting_Involved/README.rst
[DEBUG][11:48:45] /home/sanket/coala/CONTRIBUTING.md
[DEBUG][11:48:45] /home/sanket/coala/docs/Users/External_APIs.rst
[DEBUG][11:48:45] /home/sanket/coala/docs/Users/Tutorials/Writing_Bears.rst
[DEBUG][11:48:45] Running bear SpaceConsistencyBear...
[DEBUG][11:48:45] Running bear LineLengthBear...
[DEBUG][11:48:45] Running bear SpaceConsistencyBear...
```